### PR TITLE
4D load crash medInria

### PR DESCRIPTION
**Pull request by Florent2305**

### CODE REVIEW

**Comment by mathildemerle** [comment]
If possible, is it possible to clarify the title and description of this PR? It will be easier to search this PR if needed in the future :)

File: src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImageInteractor.cpp [hunk 1]
```diff
@@ -232,6 +232,8 @@ bool medVtkViewItkDataImageInteractor::SetViewInput(medAbstractData* data, int l
 
 void medVtkViewItkDataImageInteractor::initParameters(medAbstractImageData* data)
 {
+    d->imageData = data; //used among others for 4D
```
Comment (mathildemerle) [inline src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImageInteractor.cpp]: i wonder if it solves also the dicom import crash. This class did not find imageData :o

Comment (Florent2305) [reply src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImageInteractor.cpp]: About 4D Dicom?

Comment (mathildemerle) [reply src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImageInteractor.cpp]: indeed, the one i sent you this morning ^^

Comment (Florent2305) [reply src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImageInteractor.cpp]: From my side it's ok, I mean it's load without crash.
But I don't now this data and maybe it's not render correctly.

**Pull request merged**